Source/Redemption/Private/PauseMenuWidget.cpp

```diff
@@ -0,0 +1,46 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "PauseMenuWidget.h"
+#include "Components/Button.h"
+#include "Kismet/GameplayStatics.h"
+#include "Kismet/KismetSystemLibrary.h"
+
+
+void UPauseMenuWidget::Resume()
+{
+	// Hide the pause menu widget
+	SetVisibility(ESlateVisibility::Hidden);
+
+	// Unpause the game
+	UGameplayStatics::SetGamePaused(GetWorld(), false);
+
+	// Set input mode back to game
+	APlayerController* Controller = UGameplayStatics::GetPlayerController(GetWorld(), 0);
+	Controller->SetInputMode(FInputModeGameOnly());
+	Controller->bShowMouseCursor = false;
+}
+
+void UPauseMenuWidget::Exit()
+{
+	//UE_LOG(LogTemp, Warning, TEXT("START"));
+	UGameplayStatics::OpenLevel(GetWorld(), "MainMenu");
+	APlayerController* Controller = UGameplayStatics::GetPlayerController(GetWorld(), 0);
+	Controller->SetInputMode(FInputModeGameOnly());
+	Controller->bShowMouseCursor = false;
```

Comment:
since we are going to the main menu I think there is no reason to set it to game only.

Reply:
okay